sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/statsEstimation/JoinEstimation.scala

```diff
@@ -233,7 +233,8 @@ case class JoinEstimation(join: Join) extends Logging {
     val rightKeyStat = rightStats.attributeStats(rightKey)
     val maxNdv = leftKeyStat.distinctCount.get.max(rightKeyStat.distinctCount.get)
```

Comment:
This line is not needed after your change as I understand

Reply:
Removed.